lib/fog/core/model.rb

```diff
@@ -21,16 +21,7 @@ def initialize(new_attributes = {})
     end
 
     def inspect
-      Thread.current[:formatador] ||= Formatador.new
-      data = "#{Thread.current[:formatador].indentation}<#{self.class.name}"
-      Thread.current[:formatador].indent do
-        unless self.class.attributes.empty?
-          data << "\n#{Thread.current[:formatador].indentation}"
-          data << self.class.attributes.map { |attribute| "#{attribute}=#{send(attribute).inspect}" }.join(",\n#{Thread.current[:formatador].indentation}")
-        end
-      end
-      data << "\n#{Thread.current[:formatador].indentation}>"
-      data
+      Fog::Formatador.new(self)
```

Comment:
The same logic is in Fog::Formatador ( comprised of init_string(), attribute_string(), and object_attributes() ), so we're definitely being DRY by taking out the logic in model.rb. But it also goes through nested objects, which is why I was asking if a parameter to prevent that from happening in this case. It seemed clear to me that yes a param is needed to prevent nested objects from being included in the string for this case, but I figured I would see what everyone thought first. As for the param, we could say ```Fog::Formatador.new(obj, level)``` where level is an integer, though that's kind of cryptic.